### ISSUE TITLE
Dataset Creation

### ISSUE THREAD

**Issue by muneeb699** [zahan97/Bird-Species-Classification#1]
Hi Zahan,

I need to ask you, from where did you get the .csv file of bird dataset? Did you create it by yourself because i cant able to see the script?

**Comment by zahan97** [comment]
the exact .csv file is from this site [http://www.vision.caltech.edu/visipedia/CUB-200.html]

**Comment by zahan97** [comment]
the above is a newer version...
the link in the readme is the exact version used

**Comment by muneeb699** [comment]
You used dataset CUB200-2011 not CUB 200-2010. Is that right?
Second, I downloaded the whole dataset and there is no such file feature_matrix.csv. If you see README file, there is no such file in which they just mentioned the attributes for each image.

**Comment by zahan97** [comment]
CUB200-2011 is an older version...
feature_matrix.csv is what I named the file after pre-processing...

**Comment by muneeb699** [comment]
Yes, and I want to see how did you preprocess it because there is no code in the repository.
